### PR TITLE
Es localization

### PR DESCRIPTION
**Pull request by rogerpb98**

### CODE REVIEW

File: client/public/static/locales/es/common.json [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
-  "NEW_BUILD": "__STRING_NOT_TRANSLATED__",
-  "NOT_APPLICABLE": "__STRING_NOT_TRANSLATED__",
-  "NO_BONUS": "__STRING_NOT_TRANSLATED__",
+  "ACTIONS": "Acciones",
+  "ANONYMOUS": "Anonimo",
+  "AVAILABLE_AT_LEVEL": "Disponible al nivel {{level}}",
+  "BACK": "Atrás",
+  "BASE": "Base",
+  "BUFFS": "Bufos",
+  "BUFF_APPLIED": "{{count}} Bufo aplicado",
```
Comment (lightyearsaway) [inline client/public/static/locales/es/common.json:8]: Should it be "bufo" (lowercase)?

File: client/public/static/locales/es/common.json [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
+  "CONFIRM_RESTART_BUILD": "Reestablecer el conjunto eliminará todos los objetos y características de ésta build. ¿Estás seguro?",
+  "CONTRIBUTE_GITHUB": "Contribuye en GitHub",
+  "COPY_BUILD": "Duplicar conjunto",
+  "COPY_BUILD_SUCCESS": "Conjunto copiado.",
+  "COPY_LINK": "Copiar link",
+  "COPY_LINK_SUCCESS": "Link copiado al portapapeles.",
+  "CREATED": "Creado",
+  "CREATE_NEW_BUILD": "Crear nuevo conjunto",
+  "DELETE": "Eliminar",
+  "DELETE_BUILD": "Eliminar conjunto.",
```
Comment (lightyearsaway) [inline client/public/static/locales/es/common.json:28]: This is used on a button, probably best to not have the period?

File: client/public/static/locales/es/common.json [hunk 3]
```diff
@@ -1,91 +1,91 @@
...
+  "MULTIPLE_AP_EXO": "Tienes múltiples exo PA.",
+  "MULTIPLE_MP_EXO": "Tienes múltiples exo PM.",
+  "MULTIPLE_PRYSMARADITES": "Tienes múltiples prismariditas equipadas.",
+  "MULTIPLE_RANGE_EXO": "Tienes múltiples exo Alcance.",
+  "MY_BUILDS": "Mis conjuntos",
+  "MY_FAVORITES": "Mis gavoritos",
+  "NEW_BUILD": "Nueva build",
+  "NOT_APPLICABLE": "N/A",
+  "NO_BONUS": "No se ha encontrado bonus de set",
   "NO_BUILDS_MATCHED": "__STRING_NOT_TRANSLATED__",
```
Comment (lightyearsaway) [inline client/public/static/locales/es/common.json:62]: This is used here:
!(https://i.imgur.com/T4b3iMu.png)[No builds matched]

File: client/public/static/locales/es/weapon_spell_effect.json [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
-  "NO_LOS": "__STRING_NOT_TRANSLATED__",
+  "HEAL": "Curas",
+  "INCREASE_BASE_DAMAGE": "+{{damageIncrease}} daño base",
+  "LINEAR": "Lineal",
+  "MAX_STACKS_APPLIED": "Has llegado al límite de acumulaciones.",
+  "MELEE": "CaC",
+  "MODIFIABLE": "Modificable",
+  "NON_CRIT": "No critico",
+  "NO_LOS": "No requiere línea de visión",
   "NO_WEAPON_SKILL": "__STRING_NOT_TRANSLATED__",
```
Comment (lightyearsaway) [inline client/public/static/locales/es/weapon_spell_effect.json:40]: No weapon skill

File: client/public/static/locales/es/common.json (hunk 1 above)
Comment (rogerpb98) [reply client/public/static/locales/es/common.json:8]: Yes

**Pull request merged**